conftest.py

```diff
@@ -13,7 +14,14 @@
 
 try:
     import torch
-    pytorch_available = True
+
+    # If we are on windows and using Python 3.10, despite it importing correctly, PyTorch is currently broken,
+    # see https://pytorch.org/get-started/locally/ showing lack of support.
```

Comment:
has this been observed? a rather old issue https://github.com/pytorch/pytorch/issues/90530 suggests that the website just needs to be updated.

Reply:
Yeah, it isn't working for me locally and it seems to have failed [here](https://github.com/PrincetonUniversity/PsyNeuLink/actions/runs/4832641673/jobs/8611674071) in CI as well. Error is weird though. They have wheels but and things import fine but you get these Numpy not available errors. Couldn't find a work around either.

Reply:
I think we're hitting https://github.com/pytorch/pytorch/issues/78341, which was unfortunately closed without a fix.
running python 3.10 locally I see:
```
Python 3.10.11 (tags/v3.10.11:7d4cc5a, Apr  5 2023, 00:38:17) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> torch.tensor([1,2,3])
<stdin>:1: UserWarning: Failed to initialize NumPy: module compiled against API version 0x10 but this version of numpy
 is 0xf . Check the section C-API incompatibility at the Troubleshooting ImportError section at https://numpy.org/devd
ocs/user/troubleshooting-importerror.html#c-api-incompatibility for indications on how to solve this problem . (Trigge
red internally at ..\torch\csrc\utils\tensor_numpy.cpp:84.)
tensor([1, 2, 3])
```

It looks like pytorch packaging is broken. it should either build wheels against the oldest supported numpy wheels or indicate the minimum required version.